questionpy_server/api/models.py

```diff
@@ -54,20 +54,20 @@ class QuestionCreateArguments(OptionalQuestionStateHash):
     form_data: dict[str, object]
 
 
-class GradingMethod(Enum):
-    ALWAYS_MANUAL_GRADING_REQUIRED = 'ALWAYS_MANUAL_GRADING_REQUIRED'
-    AUTOMATICALLY_GRADABLE = 'AUTOMATICALLY_GRADABLE'
-    AUTOMATICALLY_GRADABLE_WITH_COUNTBACK = 'AUTOMATICALLY_GRADABLE_WITH_COUNTBACK'
+class ScoringMethod(Enum):
+    ALWAYS_MANUAL_SCORING_REQUIRED = 'ALWAYS_MANUAL_SCORING_REQUIRED'
+    AUTOMATICALLY_SCORABLE = 'AUTOMATICALLY_SCORABLE'
+    AUTOMATICALLY_SCORABLE_WITH_COUNTBACK = 'AUTOMATICALLY_SCORABLE_WITH_COUNTBACK'
 
 
 class ResponseClass(BaseModel):
-    response_class: str
-    fraction: float
+    response_class: Annotated[str, Field(max_length=30, strict=True)]
```

Comment:
Hier geht auch pydantic's `constr`. Und an einigen anderen Stellen `conint`.
```suggestion
    response_class: constr(max_length=30, strict=True)
```

Reply:
Mypy beschwert sich dann leider:
```
questionpy_server/api/models.py:25: error: Invalid type comment or annotation  [valid-type]
questionpy_server/api/models.py:25: note: Suggestion: use constr[...] instead of constr(...)
```

Ich belasse es daher mal bei Annotated/Field.